src/categories/categories.controller.ts

```diff
@@ -0,0 +1,119 @@
+import {
+    Body,
+    ConflictException,
+    Controller,
+    HttpCode,
+    HttpStatus,
+    Post,
+    UseGuards,
+    Get,
+    Query,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Query' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5693791)